css/video-autoplay.css

```diff
@@ -22,17 +22,21 @@ i-amphtml-video-mask {
  * Animated equalizer icon for muted autoplaying videos.
  */
 
-[noaudio] .amp-video-eq {
+.amp-video-eq {
   /* Don't display icon for videos flagged as having no audio. */
```

Comment:
Shouldn't this comment be updated?  It's more than just `noaudio` videos that won't have the equalizer.

Reply:
Done! And it helped me realize that The `noaudio` case was broken, so fixed. 😅